docs/quickstarts/nextjs.mdx

```diff
@@ -122,32 +118,34 @@ Select your preferred router below to learn how to make this data available acro
 Now that Clerk is installed and mounted in your application, you can decide which pages are public and which should require authentication to access:
 
 1. Create a `middleware.ts` file at the root of your project, or in your `src/` directory if you have one.
-2. In your `middleware.ts`, export Clerk's [`authMiddleware()`](https://clerk.com/docs/references/nextjs/auth-middleware) helper:
-  ```tsx filename="middleware.ts"
-  import { authMiddleware } from "@clerk/nextjs";
+1. In your `middleware.ts`, export Clerk's [`authMiddleware()`](https://clerk.com/docs/references/nextjs/auth-middleware) helper. This helper enables authentication and blocks access for signed out visitors on routes that your middleware runs on.
 
-  export default authMiddleware({});
-  ```
-3. With `authMiddleware()`, authentication will be enabled by default on all routes that your Next.js middleware runs on, blocking access to all signed out visitors. You can specify valid routes using Next.js's [`matcher`](https://nextjs.org/docs/app/building-your-application/routing/middleware#matcher). Add the following code to your `middleware.ts` to protect your entire application:
-
-  ```tsx filename="middleware.ts" {15}
-  import { authMiddleware } from "@clerk/nextjs";
+    Next.js middleware only runs on routes specified with [`matcher`](https://nextjs.org/docs/app/building-your-application/routing/middleware#matcher). Add the following code to your `middleware.ts` to protect your routes:
 
-  export default authMiddleware({
-    // Routes that can be accessed while signed out
-    publicRoutes: ['/anyone-can-visit-this-route'],
-    // Routes that can always be accessed, and have
-    // no authentication information
-    ignoredRoutes: ['/no-auth-in-this-route'],
-  });
+    ```tsx filename="middleware.ts"
+    import { authMiddleware } from "@clerk/nextjs";
 
-  export const config = {
-    // Protects all routes, including api/trpc.
     // See https://clerk.com/docs/references/nextjs/auth-middleware
     // for more information about configuring your Middleware
-    matcher: ["/((?!.+\\.[\\w]+$|_next).*)", "/", "/(api|trpc)(.*)"],
-  };
-  ```
+
+    export default authMiddleware({
+      // Allow signed out users to access the specified routes:
+      // publicRoutes: ['/anyone-can-visit-this-route'],
+      // Prevent the specified routes from accessing
+      // authentication information:
+      // ignoredRoutes: ['/no-auth-in-this-route'],

```

Comment:
Did you mean to comment ALL of this out?

```suggestion
      publicRoutes: ['/anyone-can-visit-this-route'],
      // Prevent the specified routes from accessing
      // authentication information:
      ignoredRoutes: ['/no-auth-in-this-route'],

```

Reply:
Yes, Colin requested "Can we comment out the scaffolded ignoredRoutes and publicRoutes" [here](https://clerkinc.slack.com/archives/C01QFRQNHSN/p1710219197063989)